api/default_networks.go

```diff
@@ -346,6 +347,39 @@ func baseSepolia(proxyHost, stageName string) params.Network {
 	}
 }
 
+func statusNetworkSepolia(proxyHost string) params.Network {
+	const chainID = common.StatusNetworkSepoliaChainID
+	const chainName = "status"
+	const networkName = "sepolia"
+
+	rpcProviders := []params.RpcProvider{
+		// Smart proxy provider
+		*params.NewEthRpcProxyProvider(chainID, StatusSmartProxy, smartProxyUrl(proxyHost, chainName, networkName), false),
+		// Direct providers
+		*params.NewDirectProvider(chainID, DirectStatus, "https://public.sepolia.rpc.status.network", false),
+	}
+
+	return params.Network{
+		ChainID:                chainID,
+		ChainName:              "Status Network Sepolia",
+		RpcProviders:           rpcProviders,
+		BlockExplorerURL:       "https://sepoliascan.status.network/",
+		IconURL:                "network/Network=Status-test",
+		ChainColor:             "#7140FD",
+		ShortName:              "status",
+		NativeCurrencyName:     "Ether",
+		NativeCurrencySymbol:   "ETH",
+		NativeCurrencyDecimals: 18,
+		IsTest:                 true,
+		Layer:                  2,
+		Enabled:                false,
+		IsActive:               true,
+		IsDeactivatable:        true,
+		// TODO: Update related chain ID
+		// RelatedChainID:  1,
```

Comment:
I am not sure if RelatedChainID is in use anywhere?

Reply:
That's true, but not in usage. There is no use case either in the desktop or in the mobile app for `GetEthereumChains` API endpoint.

Reply:
Mobile still uses the `GetEthereumChains` endpoint.

https://github.com/status-im/status-mobile/blob/94ef9f7cfde08b631fc4df75810635d4c56ee238/src/status_im/contexts/wallet/events.cljs#L461 

Just curious. If the desktop is not using it, How does it get chain information? If that RPC needs to be updated or something in mobile, please let us know 🙏

Reply:
Seems we're using `getFlatEthereumChains` instead of `GetEthereumChains`.

Reply:
And also `RelatedChainID` doesn't figure anywhere. Don't see any use case where we need it.

Reply:
We should clean that up a bit (not here ofc).

Reply:
There was a recent design change in Desktop where Mainnet and Testnet networks are treated more independently and we don't care anymore about, for example, Optimism Sepolia being the testnet counterpart of Optimism.
Not sure if that piece of information is gonna be required at some point, but that doesn't seem to be the case at the moment (moreover, the approach we had was a bit cumbersome to work with when handling multiple testnets, i.e. upgrading from Goerli to Sepolia)

Reply:
That's true, I don't think we will need this (`RelatedChainId`) in future, additionally, if any other networks can be added by the user, it helps to treat it independently as you said, instead of interlinking them with other chains. I guess this would make it easy for chain abstraction support in future. 

Mobile will start to move away from `GetEthereumChains`. Thanks 🙏